src/main.ts

```diff
@@ -255,31 +253,33 @@ class Benchmark extends utils.Adapter {
 		if (!this.config.secondaryMode) {
 			if (obj.command === 'test') {
 				// run all tests on test command - do not await, we want to respond to message
-				this.runTests(allTests);
+				this.runTests(Object.keys(allTests));
 			} else if (allTests[obj.command]) {
-				const selectedTests:Record<string, any> = {};
-				selectedTests[obj.command] = allTests[obj.command];
-				this.runTests(selectedTests);
+				// this.runTests({
+				// 	[obj.command]: allTests[obj.command]
+				// });
+				this.runTests([obj.command]);
```

Comment:
See other comment above. I'm not sure why you needed an object. If you do, there's no need for an intermediate assignment and you can use the commented-out syntax instead.